app/services/export/s3_uploader.rb

```diff
@@ -4,14 +4,22 @@ class S3UploadError < StandardError; end
   class S3Uploader
     def initialize(file:, filename:, use_public_bucket:)
       @use_public_bucket = use_public_bucket
+      @bucket_name_from_env_var = ENV.fetch("EXPORT_DOWNLOAD_S3_BUCKET", false)
       @config = S3UploaderConfig.new(use_public_bucket: use_public_bucket)
-      @client = Aws::S3::Client.new(
-        region: config.region,
-        credentials: Aws::Credentials.new(
-          config.key_id,
-          config.secret_key
+      @client = if @bucket_name_from_env_var
+        Aws::S3::Client.new(
+          region: "eu-west-2",
```

Comment:
I wonder if we want to put a comment here saying we have hardcoded the region since we dotn expect it to change. Or make it the default and use config.region if it exists. Setting config.region with an env var elsewhere.
This shouldnt block deployment though.